recipe/meta.yaml

```diff
@@ -1,33 +1,40 @@
-{% set version = "0.7.3" %}
+{% set name = "vispy" %}
+{% set version = "0.12.1" %}
 
 package:
   name: vispy
   version: {{ version }}
 
 source:
   fn: vispy-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/v/vispy/vispy-{{ version }}.tar.gz
-  sha256: 1edd3434247577e0910f9c76331b82a54be7e5381c022120c7336e73167b155c
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e00881c1d0f9b2c08eb6e26e9361adbde8a3a96e7878ee6c1e199e7c5872224f
   patches:
     - ctypes_fontconfig.diff
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  ignore_run_exports:
+    - fontconfig
+    - python
```

Comment:
Why do we ignore run exports of fontconfig and python? What will be if we remove `ignore_run_exports`?

Reply:
Interestingly enough, I get an overdepending error if these are not included.